Utils.py

```diff
@@ -732,6 +732,15 @@ def is_kivy_running():
         root.update()
 
 
+def askyesnocancel(title: str, text: str) -> Optional[bool]:
```

Comment:
If we're going to have this wrapper in `Utils`, I think a snake case name would be better.

And a doc string would be nice (saying that it wraps the tkinter function, and that return `True` corresponds to the "Yes" button, `False` to "No", and `None` to any other way of closing the window).

Reply:
docstring is a good call. it seemed weird to use a different name than the function it's wrapping but it probably is the correct direction.

Reply:
> it seemed weird to use a different name than the function it's wrapping

Yeah, I don't feel strongly about it because of that.